pages/common/from.md

```diff
@@ -0,0 +1,20 @@
+# from
+
+> Prints mail header lines from your mailbox.
```

Comment:
Generally we try to avoid the 2nd person (e.g. `you`, `your`, etc) in tldr-pages :slightly_smiling_face:
```suggestion
> Prints mail header lines from the current user's mailbox.
```